src/include/btmem.h

```diff
@@ -716,17 +716,23 @@ struct __wt_page {
 
     size_t memory_footprint; /* Memory attached to the page */
 
-    /* Page's on-disk representation: NULL for pages created in memory. */
-    const WT_PAGE_HEADER *dsk;
-
     /* If/when the page is modified, we need lots more information. */
     WT_PAGE_MODIFY *modify;
 
+    /* Page's on-disk representation: NULL for pages created in memory. */
+    const WT_PAGE_HEADER *dsk;
+
     /*
      * !!!
      * This is the 64 byte boundary, try to keep hot fields above here.
      */
 
+    /*
+     * The allocated memory for the page-disk image, solely used for cache tracking purposes. The
+     * 'dsk' variable above contains the actual number of bytes consumed by the disk image.
```

Comment:
I find the terminology confusing/ambiguous here. In particular, on my first reading, I interpreted "disk image" as the thing we read from disk -- which is the larger container for the actual bytes we use to construct the in memory page.

Maybe reword this as:
```
The allocated memory for the page's disk image, solely used for cache tracking purposes. The 'dsk' variable
above contains the actual size of the page contained in the disk image.
```

Reply:
Thanks, updated in [50b19cf](https://github.com/wiredtiger/wiredtiger/pull/9279/commits/50b19cf7349c635f8aaed022f121a76cde3b6680)